README.md

```diff
@@ -10,3 +10,144 @@ We recommend using this npm package in order to receive updates/fixes.
 ## Notes
 
 Use `bun install` to avoid issues.
+
+## Integration Guidance
+
+### getVersion
+
+The `getVersion` method retrieves the version information from the Algorand Ledger app.
+
+```typescript
+import { AlgorandApp } from '@zondax/ledger-algorand'
+
+const transport = ...
```

Comment:
We could add more information about transports itself, right?

Reply:
At least a reference to what ledger describes/provides as transports. Some introduction, or link to ledger documentation about it is useful.